.platform/routes.yaml

```diff
@@ -364,6 +364,13 @@ https://docs.upsun.com/:
       "/anchors/environments/cancel-activity.html": { "to": "/environments/cancel-activity.html", "code": 301, "prefix": false }
       "/anchors/admin/servers.html": { "to": "/administration/servers.html", "code": 301, "prefix": false }
 
+      # Redirects for moved Upsun GS guides
+      "/get-started/express/(.*).html": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
+      "/get-started/flask/(.*).html": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
```

Comment:
I got 404 for https://docs.upsun.com/get-started/flask/deploy/prepare.html. Shouldn't this be `/get-started/flask/deploy/(.*.html)` ?

I also tested the Upsun docs page from the CI. It is not redirecting.

Reply:
The former Get Started > Flask section had multiple subpages that may also still pop up as Google search results, so we want to tackle all those with a single redirect. That being said, my proposal was indeed wrong, I think maybe `/get-started/flask(.*)` would be better suited to address the entire section in a simple way? @chadwcarlson for review please.